src/main/java/org/sonatype/gradle/plugins/scan/nexus/iq/scan/NexusIqPluginScanExtension.java

```diff
@@ -61,7 +61,7 @@ public NexusIqPluginScanExtension(Project project) {
     organizationId = "";
     simulationEnabled = false;
     simulatedPolicyActionId = PolicyAction.NONE.toString();
-    scanFolderPath = project.getBuildDir() + File.separator + SONATYPE_CLM_FOLDER + File.separator;
```

Comment:
`import java.io.File;` becomes unused after this. It's better to remove that import

Reply:
Thanks. Removed in [fde1eac](https://github.com/sonatype-nexus-community/scan-gradle-plugin/pull/107/commits/fde1eac60cff962fe3eccb07150860370a1d8179)